widget_examples/multi_file_viewer/widgets.json

```
@@ -0,0 +1,55 @@
+{
+  "whitepapers": {
+    "type": "multi_file_viewer",
+    "name": "Whitepapers",
+    "description": "List of available whitepapers",
+    "endpoint": "/random/whitepapers/view-base64",
+    "gridData": {
+      "w": 40,
+      "h": 10
+    },
+    "params": [
+      {
+        "type": "endpoint",
+        "paramName": "whitepaper",
+        "value": "http://localhost:5011/random/whitepapers",
+        "label": "Whitepaper",
+        "description": "Whitepaper to display.",
+        "optionsEndpoint": "/random/whitepapers",
+        "show": false,
+        "optionsParams": {
+          "type": "$type"
+        }
+      },
+      {
+        "type": "text",
+        "paramName": "type",
```

```suggestion
        "paramName": "category",
```
(Basing this on @montezdesousa's [comment](https://github.com/OpenBB-finance/backend-examples-for-openbb-workspace/pull/17#issuecomment-2704108956)).